scripts/build.py

```diff
@@ -1102,10 +1102,22 @@ def build():
     parser = optparse.OptionParser()
     parser.add_option(
         '--prod_env', action='store_true', default=False, dest='prod_mode')
+    parser.add_option(
+        '--minify_third_party_libs_only', action='store_true', default=False,
+        dest='minify_third_party_libs_only')
     options = parser.parse_args()[0]
     # Regenerate /third_party/generated from scratch.
     safe_delete_directory_tree(THIRD_PARTY_GENERATED_DEV_DIR)
     build_third_party_libs(THIRD_PARTY_GENERATED_DEV_DIR)
+
+    # If minify_third_party_libs_only is set to True, skips the rest of the
+    # build process once third party libs are minified.
+    if options.minify_third_party_libs_only:
+        if not options.prod_mode:
+            raise Exception(
+                'minify_third_party_libs_only is set to true in non-prod mode.')
```

Comment:
Just for clarity, could you please change "is set to true" --> "should not be set"? I think that's more normative and makes it easier for the developer to figure out what to fix.

Reply:
Good point, thanks!